docs/moderator-onboarding-guide.md

```diff
@@ -0,0 +1,49 @@
+
+## The Forum
+
+### First Steps
+
+The first thing you may notice after being given moderator status on the forum is that your interface will look somewhat different with new admin tools to explore and access to the Mod-Team subforum as well.
+
+Some of the new tools will appear inside a new menu item that looks like a wrench. Some will appear as new tabs or buttons or even new enabled options within the forum menus.
+
+To get familiar with the new tools and powers you can combine one or more of the following methods during your first week with this elevated role. (The first two of which are the most important.)
+
+### Become Familiar with the Discourse Admin Tools
+
+The freeCodeCamp forum is actually a Discourse forum and follows many of the same guidelines of other forums built on Discourse. To begin to get familiar with Discourse and the moderation role, start by reading Discourse's [new user guide](https://meta.discourse.org/t/discourse-new-user-guide/96331) then read Discourse's [new moderator guide](https://meta.discourse.org/t/discourse-moderation-guide/63116).
+
+### Shadow a Mod
+
+All moderator actions can be followed by reviewing the [action logs](https://forum.freecodecamp.org/admin/logs/staff_action_logs). The actions taken by automated tools like 'akistmet' or 'system' can mostly be ignored until they result in a post that needs to be reviewed. Posts to be reviewed will show up in the [Review](https://forum.freecodecamp.org/review) area of the forum.
+
+For the first week or so you will want to pay attention to what is getting flagged and what is being reviewed and compare that to the actions being taken upon the flagged posts. So you may see the system account flag a post because the user created it too quickly. In many cases, the moderators will unflag the post by clicking on the "Approve Post" button or mark it as "Not Spam" (depending on the flag type).
+
+Commonly, spam flags can also be raised by members or moderators. Common duplicitious behaviour would involve opening an account, making a post that seems harmless, then editing that post later on to add a link to an external site for the purpose of advertising it. In this case, usually the member account is fairly new and has only made this one post thus far, which indicates that the account was opened for the sole purpose of spamming the community. The decision should be made to delete the account after the first offence in this case. The same applies for accounts whose first post is deemed to be spam.
+
+You may notice moderators performing a procedure called 'split topic'. Which can be interpreted as either a moderator has split a post that was made in error into the tail-end of a topic that is aleady closed into its own thread for eg.; or a moderator merged duplicate topics that a single user has created for the same fCC challenge. Watching this procedure will highlight different actions and their causes.
```

Comment:
```suggestion
You may notice moderators performing a procedure called 'split topic'. This may be a case where a moderator has split a post that was made erroneously on an existing topic into a new topic, or a moderator merged duplicate topics that a single user has created for the same question. Watching this procedure will highlight different actions and their causes.
```

This one was tricky to read, IMHO.

Reply:
yes I believe it! It's almost a reflection of my own turmoil!